src/pages/Invoice/CreateInvoice.js

```diff
@@ -187,7 +188,7 @@ const CreateInvoice = () => {
                                         {" "}
                                         <FormattedMessage id="invoice.issuedAt" />
                                     </label>
-                                    <input
+                                    <Input
                                         className="input-sm"
```

Comment:
make new props `size` can support `sm` | `md` for now if `lg` require will introduce later

Reply:
`md` being default

Reply:
introduces size props

Reply:
make md as default class